### PR TITLE
update to 1.20

### LINKED ISSUE #10: About 1.19.4 and 1.20 update
Is an update to these versions planned?

If not, can i ask on Fabric's Discord if someone is interested in porting themselves?

### PR DESCRIPTION
**Pull request by DerpDerpling**
*Maintainer edit:*

Closes #10

### CODE REVIEW

File: src/main/resources/fabric.mod.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
   "depends": {
     "fabricloader": ">=0.14.9",
     "fabric": "*",
-    "minecraft": "1.19.*",
+    "minecraft": ">=1.20-",
```
Comment (Jummit) [inline src/main/resources/fabric.mod.json:32]: I haven't seen this yet, what does the minus in the version number mean?

**Review by Jummit — APPROVED** [review]
Thanks for the port!

File: src/main/resources/fabric.mod.json (hunk 1 above)
Comment (DerpDerpling) [reply src/main/resources/fabric.mod.json:32]: the - supports any version of 1.20, as in 1.20.1, and 1.20-rc1 etc

Comment (Jummit) [reply src/main/resources/fabric.mod.json:32]: Cool!

**Review by Jummit — APPROVED** [review]

**Comment by Jummit** [comment]
Oops, this should be merged into the 1.20 branch. Could you rebase?

**Comment by DerpDerpling** [comment]
> Oops, this should be merged into the 1.20 branch. Could you rebase?

I’m somewhat new to GitHub pull requests, what is rebasing?

**Comment by Jummit** [comment]
> > Oops, this should be merged into the 1.20 branch. Could you rebase?
> 
> I’m somewhat new to GitHub pull requests, what is rebasing?

I actually have never done this myself, but is it possible to switch the branch this PR is targeting?

Rebasing is a Git concept where you apply a branch ontop of another one: [some confusing documentation page](https://git-scm.com/book/en/v2/Git-Branching-Rebasing)

You should be able to do something like `git rebase 1.20` after pulling the 1.20 branch from upstream.

**Comment by DerpDerpling** [comment]
> Rebasing is a Git concept where you apply a branch ontop of another one: [some confusing documentation page](https://git-scm.com/book/en/v2/Git-Branching-Rebasing)
> 
> You should be able to do something like `git rebase 1.20` after pulling the 1.20 branch from upstream.

was able to fix it by changing the selected branch on the github website lol

**Comment by DerpDerpling** [comment]
fixed the conflicts, should be ready to merge

**Pull request merged**